app/lib/payment_gateways/stripe_crypt.rb

```diff
@@ -36,21 +32,41 @@ def customer
       @customer ||= find_or_create_customer
     end
 
+    def update_billing_address(billing_address)
+      begin
+        Stripe.api_key = api_key  # Set actual Stripe secret key
+        latest_payment_method_id = latest_payment_method_id_for_customer
+        return true unless latest_payment_method_id.present?
+        payment_method = Stripe::PaymentMethod.retrieve(payment_method_id)
```

Comment:
I'm getting an error here when trying to update the address:
![image](https://github.com/3scale/porta/assets/1270328/d7dcd12d-f593-4034-9c96-87e29b4c4198)

Reply:
ummmm I have not received this error but let me try to reproduce it.

Reply:
@mayorova Yes due to code refactor id name was passed wrong

I have fixed it and now it works well

![image](https://github.com/3scale/porta/assets/84672731/62bfbd9a-f98c-478b-b0c5-a7f726744eae)
![Uploading image.png…]()